postgres/datadog_checks/postgres/postgres.py

```diff
@@ -716,6 +718,7 @@ def check(self, _):
             if self._config.dbm_enabled:
                 self.statement_metrics.run_job_loop(tags)
                 self.statement_samples.run_job_loop(tags)
+                self.metadata_samples.run_job_loop(tags)
```

Comment:
Not saying anything new, but we are adding yet another open connection for the datadog agent for this job loop. Let's make sure this is the last one and this class is used for all additional data not collected as part of the collection interval.